design/sketch.webidl

```diff
@@ -5,6 +5,17 @@ typedef long u32;
 // ****************************************************************************
 
 // Buffer
+
+callback WebGPUMappedMemorySuccessCallback = void (ArrayBuffer);
+callback WebGPUMappedMemoryErrorCallback = void (DOMString);
```

Comment:
`Error` (https://w3ctag.github.io/promises-guide/#reasons-should-be-errors)

Reply:
Done